app/Http/Controllers/OrderController.php

```diff
@@ -6,11 +6,69 @@
 use Illuminate\Database\Eloquent\ModelNotFoundException;
 
 use App\Http\Requests;
+use App\Http\Requests\OrderRequest;
 use App\Models\Order;
+use App\Models\OrderDetail;
+use App\Models\Product;
+use App\Models\Supplier;
+use Auth;
 
 class OrderController extends Controller
 {
     /**
+     * Show the form for creating a new order.
+     *
+     * @return \Illuminate\Http\Responsed
+     */
+    public function create()
+    {
+        $suppliers = Supplier::all();
+        return view('orders.create', [
+            'suppliers' => $suppliers
+        ]);
+    }
+
+    /**
+     * Store a newly created order in storage.
+     *
+     * @param \Illuminate\Http\Request $request request
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function store(OrderRequest $request)
```

Comment:
[rank A] I don't think this function is good. It's so heavy for performance. Please thinking about the way can improve it.